third-culture-software/bhima
# Update yarn dependencies

Over time, our yarn dependencies have gathered some unneeded, duplicate dependencies.  This PR updates the dependencies in the yarn.lock file to clean that up.  

This PR also disables the protractor end-to-end tests from the 'yarn test' test script.   This will be updated when we have migrated to a new framework for end-to-end tests.

### TESTING

- run 'yarn' to update the dependencies
- run 'yarn check' to verify that redundant dependencies have been removed.
- run 'yarn test' to verify that the tests still run on your system
- Try doing running the GUI (with bhima_test)
   - yarn build:db
   - yarn dev
   - try some test operations
